dockernet/src/start_relayers.sh

```diff
@@ -20,11 +20,11 @@ for chain_id in ${HOST_CHAINS[@]}; do
     printf "STRIDE <> $chain_id - Adding relayer keys..."
     $relayer_exec rly keys restore stride $RELAYER_STRIDE_ACCT "$mnemonic" >> $relayer_logs 2>&1
     $relayer_exec rly keys restore $chain_name $account_name "$mnemonic" >> $relayer_logs 2>&1
-    echo "Done"
+    echo "Done restoring relayer keys"
 
     printf "STRIDE <> $chain_id - Creating client, connection, and transfer channel..." | tee -a $relayer_logs
     $relayer_exec rly transact link stride-${chain_name} >> $relayer_logs 2>&1
-    echo "Done"
+    echo "Done creating client, connection, and transfer channel"
```

Comment:
I'd revert these to just "Done" cause they print out on one line like:

`STRIDE <> $chain_id - Adding relayer keys...Done`

Reply:
done